sdk/Jenkinsfile

```diff
@@ -1,63 +1,41 @@
 #!/usr/bin/groovy
 // Copyright (C) Pelagicore AB 2018
 
-void shutdownVagrant() {
-    // In case the destroy step fails (happens sometimes with hung up ssh connections)
-    // we retry a couple of times to make sure it shuts down and is destroyed.
-    retry(5) {
-        sh "vagrant destroy -f"
-    }
-}
-
-// Helper function to run commands through vagrant
-void vagrant(String command) {
-    sh "vagrant ssh -c \"${command}\""
-}
-
 void verifySDK(String upstream, String variant) {
 
-    try {
-        stage("Checkout repo") {
-            // Checkout the git repository and refspec pointed to by jenkins
-            checkout scm
-            // Update the submodules in the repository.
-            sh 'git submodule update --init'
-        }
-
-        stage("Start Vagrant ${variant}") {
-            // Start the machine and provision it
-            shutdownVagrant()
-            sh "vagrant up"
-        }
-
-        String sdkDir = "/home/yoctouser/sdk"
-        String archive_directory = "artifacts_${variant}"
-        String sdkFilePattern = "pelux-glibc-*.sh"
+    stage("Checkout repo") {
+        // Checkout the git repository and refspec pointed to by jenkins
+        checkout scm
+        // Update the submodules in the repository.
+        sh 'git submodule update --init'
+    }
 
-        stage("Copying latest SDK ${variant}") {
-            sh "rm -rf ${archive_directory}"
+    String sdkDir = "/home/yoctouser/sdk"
+    String archive_directory = "artifacts_${variant}"
+    String sdkFilePattern = "pelux-glibc-*.sh"
 
-            copyArtifacts(projectName: "${upstream}",
-                          selector: lastSuccessful(),
-                          filter: "artifacts_${variant}/${sdkFilePattern}")
-        }
+    stage("Copying latest SDK ${variant}") {
+        sh "rm -rf ${archive_directory}"
 
+        copyArtifacts(projectName: "${upstream}",
+                      selector: lastSuccessful(),
+                      filter: "artifacts_${variant}/${sdkFilePattern}")
+    }
+    def customImage = docker.image("pelux/pelux-yocto:yoctouser")
+    customImage.inside('-v $WORKSPACE:/workspace -v /var/yocto-cache:/var/yocto-cache --cap-add=NET_ADMIN --device=/dev/net/tun') {
```

Comment:
can't we get it from Jenkins job parameter?

Reply:
What exactly? Image name?
Multibranch jobs are not parametrized. We can add it as a parameter but in the script (and"pelux/pelux-yocto:yoctouser" will be default value)